src/fs_store.rs

```diff
@@ -452,7 +498,11 @@ impl Chunks {
         for chunk in self.incrementals.values() {
             bytes.extend(chunk);
         }
-        automerge::Automerge::load(&bytes)
+
+        automerge::Automerge::load_with_options(
+            &bytes,
+            automerge::LoadOptions::new().on_partial_load(automerge::OnPartialLoad::Ignore),
```

Comment:
This change seems suspect. What's the rationale here?

Reply:
> In this case we would expect each file to contain a full chunk

In this case it's actually, "we've put together all the chunks, but we don't have a full doc for whatever reason", no?

Reply:
Oh, okay, that makes it even worse. Will revert this line.